addons/fix-pasted-scripts/userscript.js

```diff
@@ -0,0 +1,45 @@
+// This addon works by modifying the VM and Blockly to not react to clicking scripts.
+
+export default async function ({ addon, global, console }) {
+  const vm = await addon.tab.traps.vm;
+  const originalBlocklyListen = await vm.editingTarget.blocks.constructor.prototype.blocklyListen;
+  const BlocklyInstance = await addon.tab.traps.getBlockly();
+  const originalObject = await BlocklyInstance.BlockSvg.prototype.onMouseDown_;
+
+  // Necessary for the CTRL + click option to work
+  var ctrlKeyPressed = false;
+  document.addEventListener("keydown", function (e) {
+    if (e.ctrlKey) {
+      ctrlKeyPressed = true;
+    }
+  });
+  document.addEventListener("keyup", function (e) {
+    if (!e.ctrlKey) {
+      ctrlKeyPressed = false;
+    }
+  });
```

Comment:
doing this doesn't feel right... can you check for e.metaKey in onMouseDown_?

Reply:
The CTRL check is actually in `newBlocklyListen(e)`, and unfortunately, `metaKey` isn't a property of the `e` passed into that function.